### ISSUE TITLE
"Failed to edit image" Tags not getting applied?

### ISSUE THREAD

**Issue by OkyDooky** [deckerst/aves#649]
**Describe the bug**
I tried to add tags on some images and the app refused to do so, today. I don't know why this is. In a cluster of images I applied a tag to, only one successfully applied.

**To Reproduce**
1. Go to thunbnail view
2. Select one or more images
3. Apply one or more tags
5. (In image view, no error message appears)

**Expected behavior**
Tag(s) should be applied like normal.

**System information and logs:**
Package: deckers.thibault.aves.libre
Installer: dev.imranr.obtainium
Aves version: 1.8.7-libre, build 9802
Flutter: stable 3.10.2
Android version: 13, API 33
Android build: lineage_ocean-userdebug 13 TQ1A.230205.002 eng.root.20230311.065633 dev-keys
Device: motorola moto g(7) power
Geocoder: ready
Mobile services: not available
System locales: en_US
Storage volumes: /storage/emulated/0/, /storage/3062-3833/
Storage grants: /storage/emulated/0/Movies/, /storage/3062-3833/Movies/, /storage/emulated/0/Pictures/, /storage/emulated/0/Documents/, /storage/emulated/0/DCIM/, /storage/3062-3833/Pictures/, /storage/emulated/0/Download/Pixel 4 Downloads and Photos/, /storage/emulated/0/Download/Image Converter/
Error reporting: false
[aves-logs-20230527_180913.txt](https://github.com/deckerst/aves/files/11583350/aves-logs-20230527_180913.txt)

**Additional context**
I had something like this happen once before, but I didn't notice an error message and it wound up corrupting my image.

**Comment by deckerst** [comment]
There's an issue with Aves v1.8.7. It also affects other aspects, like scanning WEBP for example.
I intend to release a new version soon.

**Comment by deckerst** [comment]
Can you try the [latest release](https://github.com/deckerst/aves/releases/tag/v1.8.8) and let me know if you still have the issue?

**Comment by OkyDooky** [comment]
Tried it and it seems to work, so far! Thank you for fixing bugs so quickly.
-------------
Also, unrelated to this issue, thank you for continuing to keep the Flutter version used up to date, as well. On my Galaxy Note 3, Lineage 18.1 (Android 11) is the most up to date official version and it has some strong issues with Flutter apps causing horrible visual artifacts, like choppy flashes of bands of static or partial repeats of other visual elements in the given app. Everything still functions, but it makes it a difficult experience. However, it seems to be getting fixed, since I noticed the last couple releases of Aves not experiencing those issues, so far.
